Content.Server/DeviceNetwork/Systems/NetworkConfiguratorSystem.cs

```diff
@@ -77,10 +85,17 @@ private void TryAddNetworkDevice(EntityUid? targetUid, EntityUid userUid, Networ
         if (!targetUid.HasValue || !Resolve(targetUid.Value, ref device, false))
             return;
 
-        if (string.IsNullOrEmpty(device.Address))
+        var address = device.Address;
+        if (string.IsNullOrEmpty(address))
         {
-            _popupSystem.PopupCursor(Loc.GetString("network-configurator-device-failed", ("device", targetUid)), Filter.Entities(userUid));
-            return;
+            if (MetaData(targetUid.Value).EntityLifeStage == EntityLifeStage.MapInitialized)
+            {
+                _popupSystem.PopupCursor(Loc.GetString("network-configurator-device-failed", ("device", targetUid)),
+                    Filter.Entities(userUid));
+                return;
+            }
+
+            address = $"UID: {targetUid.Value.ToString()}";
         }
```

Comment:
this is extremely sussy and im not entirely sure how its even solving what its supposed to solve

Reply:
this is because device lists store entity UIDs instead of addresses for its 'filter broadcast' event, so technically this *works*, because it will serialize the entity UIDs instead of the addresses - however, since addresses don't exist before map init, and it still does the check, i added in the entity UID as a way to easily identify which entity is which in the list for a mapper